### ISSUE TITLE
`eslint-comments/no-unused-disable` false positive when disabling line with `eslint-comments/no-unused-disable`

### ISSUE THREAD

**Issue by DetachHead** [eslint-community/eslint-plugin-eslint-comments#130]
```ts
// eslint-disable-next-line eslint-comments/no-unlimited-disable
// eslint-disable-next-line
```
errors:
```
'eslint-comments/no-unlimited-disable' rule is disabled but never reported.eslint (eslint-comments/no-unused-disable)
Unexpected unlimited 'eslint-disable-next-line' comment. Specify some rule names to disable. (eslint-comments/no-unlimited-disable)
```

original issue: https://github.com/mysticatea/eslint-plugin-eslint-comments/issues/77

**Comment by DetachHead** [comment]
@MichaelDeBoey could this please be re-opened? i did not receive a notification that the stale bot was awaiting a response (see #164)

**Comment by refex** [comment]
i have the same problem
